source/testing_govuk_pay/index.html.md.erb

```diff
@@ -23,25 +23,29 @@ Your test account type depends on your chosen payment service provider (PSP).
 | Stripe                         | Stripe test account  |
 | Worldpay                       | Sandbox test account |
 
-You're limited to one test account per GOV.UK Pay service. For example, if you have a sandbox test account and request a Stripe test account, the Stripe account will replace the sandbox account. You will lose your test data and API keys.
+You're limited to one test account per GOV.UK Pay service. For example, if you have a sandbox test account, you cannot request a Stripe test account on the same GOV.UK Pay service.
 
-### Request a Stripe test account
-
-If you created your GOV.UK Pay service before September 2024 and chose Stripe as your PSP, you may have a sandbox test account. You can swap this for a Stripe test account. 
+### Get a Stripe test account
 
 You can use a Stripe test account to see how the following work:
 
 * [transaction fees](/reporting/#psp-fees)
 * payments to your bank account
 
-To request a Stripe test account, you need to use the admin tool. 
+You already have a Stripe test account if you created your GOV.UK Pay service after September 2024 and chose Stripe as your PSP.
+
+You may have a sandbox test account if you created your GOV.UK Pay service before September 2024 and chose Stripe as your PSP.
+
+It is not possible to swap a sandbox test account for a Stripe test account within the same GOV.UK Pay service. To get a Stripe test account, you need to create a new GOV.UK Pay service.
 
 1. [Sign in to the GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services).
-1. Select Test account (sandbox) for your service. 
-1. From the Dashboard page, select Test Stripe’s reporting functionality and follow the instructions.
+1. Select **Add a new service**. 
```

Comment:
Should this be numbered 1 - 4?

Reply:
Our template automatically sets numbering if you leave every step as `1.`. It makes it a tiny tiny bit easier to insert steps at a later date because you don't need to renumber every step.

Reply:
TIL!